web/api/web_api_v1.c

```diff
@@ -934,10 +941,285 @@ inline void host_labels2json(RRDHOST *host, BUFFER *wb, size_t indentation) {
     netdata_rwlock_unlock(&host->labels.labels_rwlock);
     rrdhost_unlock(host);
 }
+#endif /* ENABLE_JSONC */
 
 extern int aclk_connected;
-inline int web_client_api_request_v1_info_fill_buffer(RRDHOST *host, BUFFER *wb)
+#ifdef ENABLE_JSONC
+#define JSON_ADD_STRING(name, str, obj)                                                                                \
```

Comment:
As I already wrote above, we have the same definition here.

Reply:
moved to common file